src/Tests/RegistrationTest.php

```diff
@@ -28,9 +28,36 @@ class RegistrationTest extends WebTestBase {
    * Test the standard registration process when there is no existing user.
    */
   public function testNormalNone() {
-    // @todo: Change the site config to disable acquisitions.
```

Comment:
Would be good to convert the @todos into comments describing what we're about to test.